lib/tokenizers/tokenizer.ex

```diff
@@ -65,31 +65,18 @@ defmodule Tokenizers.Tokenizer do
       even if `:use_cache` is false. By default it uses `:filename.basedir/3` to get
       a cache dir based in the "tokenizers_elixir" application name.
 
-    * `:additional_special_tokens` - A list of special tokens to append to the tokenizer.
-      Defaults to `[]`.
-
-    * `:padding` - Override for padding configuration. Currently the only supported
-      value is `:none` to disable padding. By default the configuration is restored
-      from the file.
-
-    * `:truncation` - Override for truncation configuration. Currently the only supported
-      value is `:none` to disable truncation. By default the configuration is restored
-      from the file.
```

Comment:
I added these options in a previous PR, because I missed we do expose the API for changing these.